_posts/2024-09-26-managing-connector-offsets.md

```diff
@@ -0,0 +1,271 @@
+---
+layout: post
+title:  "Managing Connector Offsets"
+date: 2024-08-26
+author: kate_stanley
+---
+
+When building data pipelines using Kafka Connect, or replicating data using MirrorMaker, offsets are used to keep track of the flow of data.
+Sink connectors use Kafka's standard consumer offset mechanism, while source connectors store offsets in a custom format within a Kafka topic.
+
+To manage connector offsets, rather than directly interacting with the underlying Kafka topics, you can make use of the following endpoints from the Connect REST API:
```

Comment:
Should we briefly mention common use cases for listing, altering, resetting offsets? Such as skipping a poison pill, replaying records, or specifying a specific offset to start mirroring.